src/CoreBundle/private/library/classes/TCMSResourceCollection.class.php

```diff
@@ -297,7 +298,12 @@ protected function CreateCSSResourceCollectionFile($sFileMD5, $aCSS)
 
                             $minify = ServiceLocator::getParameter('chameleon_system_core.resources.enable_external_resource_collection_minify');
                             if ($minify) {
-                                $sSubString = CssMin::minify($sSubString);
+                                // TODO use a service here (see MinifyJsJshrinkService) - if there are any problems with customer css content with new lib?
+
+                                $minifier = new CSS();
+                                $minifier->add($sSubString);
+
+                                $sSubString = $minifier->minify();
```

Comment:
We should replace the variable name here to our standard: $substring (remove the hungarian notation)

Reply:
(That variable is not introduced here. Changing it would change a number of unaffected lines. The whole method is using hungarian notation.)

Reply:
ok